packages/strapi-plugin-users-permissions/config/queries/bookshelf.js

```diff
@@ -0,0 +1,229 @@
+const _ = require('lodash');
+
+module.exports = {
+  find: async function (params) {
+    return await this
+      .forge()
+      .fetchAll({
+        withRelated: this.associations.map(x => x.alias)
+      });
+  },
+
+  count: async function (params) {
+    return await this
+      .forge()
+      .count();
+  },
+
+  findOne: async function (params) {
+    const record = await this
+      .forge({
+        [this.primaryKey]: params[this.primaryKey]
+      })
+      .fetch({
+        withRelated: this.associations.map(x => x.alias)
+      });
+
+    return record ? record.toJSON() : record;
+  },
+
+  create: async function (params) {
+    const entry = await this
+      .forge()
+      .save(Object.keys(params.values).reduce((acc, current) => {
+      if (this._attributes[current].type) {
```

Comment:
What happens if `this._attributes[current]` doesn't exist? Use _.get() to avoid that case ;)